weatherman.py

```diff
@@ -0,0 +1,80 @@
+"""
+Generating reports of temperature from given text files.
+"""
+
+import os
+import argparse
+import sys
+
+file_list = os.listdir('./weatherdata/')
+hottest_day = {}
```

Comment:
it does not store day rather stores temperature. Choose better naming.

Reply:
Changed to max_temp_. Added the extra '_' to avoid confusion, as suggested by PEP-8, with the 'max_temp' variable. Should I change this?
Also changed coldest_day and hottest_date.